pymeasure/display/widgets/image_widget.py

```
@@ -40,11 +40,12 @@ class ImageWidget(TabWidget, QtWidgets.QWidget):
     """
 
     def __init__(self, name, columns, x_axis, y_axis, z_axis=None, refresh_time=0.2,
-                 check_status=True, parent=None):
+                 check_status=True, selectors: list = [], parent=None):
```

Ruff (0.15.10)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)